Aztec/Classes/Libxml2/DOMString.swift

```diff
@@ -341,6 +341,11 @@ extension Libxml2 {
         
         // Apply Styles
         
+        fileprivate func apply(element: StandardElementType, spanning range: NSRange, attributes: [Attribute] = []) {
+            performAsyncUndoable { [weak self] in
```

Comment:
The problem if you try to centralize it here, is that you can't call this method from methods that already queued the operation (or you can, but you'd be queueing twice).

The class was designed so that only public methods would queue, an all internal operations would just be synchronous.  This is to avoid double queueing in a very simple and clear way.

I'd prefer us to stick to the existing architecture for now, and if we want to change this, do it separately.